### ISSUE TITLE
T2 Example code does not run.

### ISSUE THREAD

**Issue by OR13** [tessel/rfid-pn532#32]
Attempting to follow these instructions:

https://tessel.github.io/t2-start/modules/rfid.html

```
INFO Looking for your Tessel...
INFO Connected to athena.
INFO Tessel [athena] CLI version: 0.0.28
INFO Tessel [athena] Firmware version: 0.0.14
INFO Tessel [athena] Node version: 4.4.3
```

Hardware is connected properly, running code:

```
// Any copyright is dedicated to the Public Domain.
// http://creativecommons.org/publicdomain/zero/1.0/

/*********************************************
This basic RFID example listens for an RFID
device to come within range of the module,
then logs its UID to the console.
*********************************************/

var tessel = require('tessel');
var rfidlib = require('rfid-pn532');

var rfid = rfidlib.use(tessel.port['A']);

rfid.on('ready', function (version) {
  console.log('Ready to read RFID card');

  rfid.on('data', function(card) {
    console.log('UID:', card.uid.toString('hex'));
  });
});

rfid.on('error', function (err) {
  console.error(err);
});
```

### Expected

Expect card uid to be logged to console.

### Observed

Orange LED indicator flashes in the presence of the card, and is steady when no card is present.
Nothing is logged to console.

**Comment by OR13** [comment]
note version is undefined in 'ready' callback.

**Comment by johnnyman727** [comment]
@OR13 this was fixed this week with https://github.com/tessel/t2-firmware/pull/214.

It's expected to be released within the next few days as version `0.0.15`. Of course, you're welcome to build and flash firmware yourself [with these instructions](https://github.com/tessel/t2-firmware#compiling).

I'm not sure if the `version` returned in the `ready` callback will be fixed with this upcoming firmware patch. That might be an issue with this driver.

**Comment by johnnyman727** [comment]
Closing this as it was fixed in the last version of firmware.
